easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-3.0.1-foss-2016a.eb

```diff
@@ -0,0 +1,39 @@
+name = 'OpenFOAM'
+version = '3.0.1'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://downloads.sourceforge.net/foam/%(version)s']
+sources = [
+    SOURCE_TGZ,
+    'ThirdParty-%(version)s.tgz',
+]
+
+patches = [
+    'OpenFOAM-%(version)s_cleanup.patch',
+    'OpenFOAM-%(version)s_libreadline.patch',
+    ('ThirdParty-%(version)s_cleanup.patch', ".."),  # patch should not be applied in OpenFOAM subdir
+]
+
+dependencies = [
+    ('libreadline', '6.3'),
+    # OpenFOAM requires 64 bit METIS using 32 bit indexes (array indexes)
+    ('METIS', '5.1.0', '-32bitIDX'),
+    ('ncurses', '6.0'),
+    ('SCOTCH', '6.0.4'),
+    ('Boost', '1.60.0'),
```

Comment:
@iotaka bump to 1.61.0? cfr. #3126 

you can update the easyconfig locally, and then use `eb --update-pr 3131 <path/to>/OpenFOAM-3.0.1-foss-2016a.eb`